pom.xml

```
@@ -7,8 +7,7 @@
     <parent>
         <groupId>com.rbkmoney</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.1.1.RELEASE</version>
-        <relativePath/>
+        <version>2.3.0.RELEASE-2</version>
```

Давай сразу на эту версию
```suggestion
        <version>2.3.1.RELEASE</version>
```